task/aws/resources/resource_bucket.go

```diff
@@ -78,54 +78,17 @@ func (b *Bucket) Update(ctx context.Context) error {
 }
 
 func (b *Bucket) Delete(ctx context.Context) error {
-	listInput := s3.ListObjectsV2Input{
+	input := s3.DeleteBucketInput{
```

Comment:
Are we sure about this? As far as my experience says we can not delete non empty buckets having the remove the content first (as done prev)

Reply:
@DavidGOrtega, the whole point of this pull request is deleting objects using `rclone` instead of cloud–specific methods. We're not deleting buckets without deleting objects first (that's only possible on `az` and causes consistency issues).